app/routes/_gcn.circulars.correction.$circularId.tsx

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import type { SEOHandle } from '@nasa-gcn/remix-seo'
+import type { LoaderFunctionArgs } from '@remix-run/node'
+import { useLoaderData } from '@remix-run/react'
+
+import { getUser } from './_gcn._auth/user.server'
+import { CircularEditForm } from './_gcn.circulars.edit.$circularId/CircularEditForm'
+import { formatAuthor } from './_gcn.circulars/circulars.lib'
+import { get } from './_gcn.circulars/circulars.server'
+import type { BreadcrumbHandle } from '~/root/Title'
+
+export const handle: BreadcrumbHandle & SEOHandle = {
+  breadcrumb: 'Correction',
+  getSitemapEntries: () => null,
+}
+
+export async function loader({
+  params: { circularId },
+  request,
+}: LoaderFunctionArgs) {
+  if (!circularId) throw new Response(null, { status: 404 })
+
+  const user = await getUser(request)
+  if (!user) throw new Response(null, { status: 403 })
+  const isAuthenticated = user !== undefined
+  const circular = await get(parseFloat(circularId))
+  const data = {
+    formattedContributor: user ? formatAuthor(user) : '',
+    defaultBody: circular.body,
+    defaultSubject: circular.subject,
+    circularId: circular.circularId,
+    submitter: circular.submitter,
+    searchString: '',
+  }
+  return { isAuthenticated, data }
```

Comment:
We don't need to return `isAuthenticated`, do we? If the user is not authenticated, then the loader will throw a 403 response.

Reply:
Nope! another hold over from some previous iteration. I will remove it